### ISSUE TITLE
Changes not showing in gutter

### ISSUE THREAD

**Issue by nathanielfink** [microsoft/vscode#121564]
Issue Type: <b>Bug</b>

Hi.  I have two computers (Ubuntu Linux) I use vscode on for development. I edit the same files on both. However, on one, any changes I make show up in the gutter with blue vertical lines. On the other, they don't appear in regular editing (though they do if I open the files in the Source Control screen).

How can I fix this?

VS Code version: Code 1.55.1 (08a217c4d27a02a5bcde898fd7981bda5b49391b, 2021-04-07T15:01:24.315Z)
OS version: Linux x64 5.8.0-48-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 4500U with Radeon Graphics (6 x 1306)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 0|
|Memory (System)|15.12GB (7.53GB free)|
|Process Argv|--no-sandbox --unity-launch --crash-reporter-id 2303a9b6-a4a3-496a-bd39-1d18415bde87|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|xubuntu|
|XDG_CURRENT_DESKTOP|XFCE|
|XDG_SESSION_DESKTOP|xubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (36)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.1.0
vscode-database|baj|2.2.3
vscode-intelephense-client|bme|1.6.3
npm-intellisense|chr|1.3.1
path-intellisense|chr|2.3.0
gitignore|cod|0.6.0
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|2.1.20
git-extension-pack|don|0.1.3
githistory|don|0.6.16
es7-react-js-snippets|dsz|3.1.1
gitlens|eam|11.4.1
vscode-npm-script|eg2|0.3.20
prettier-vscode|esb|6.3.2
vscode-express-snippets|Exp|1.1.1
php-debug|fel|1.14.12
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
docker-linter|hen|0.5.0
crontab-syntax-highlight|hog|0.0.1
beautify|Hoo|1.5.0
githd|hui|2.2.4
search-node-modules|jas|1.3.0
node-module-intellisense|jhe|1.6.0
ftp-sync|luk|0.3.9
mongodb-vscode|mon|0.5.0
vscode-docker|ms-|1.12.1
python|ms-|2021.3.680753044
jupyter|ms-|2021.5.745244803
autotrim|Nat|1.0.6
nodejs-snippets|piy|0.0.2
php-ci|sma|0.4.2
nodejs-extension-pack|wad|0.1.9
delete-node-modules|Wsc|1.0.6
JavaScriptSnippets|xab|1.8.0
vscode-open-in-github|ziy|1.3.6

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt678:30270856
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvsdeb440:30248342
pythonvsded773:30248341
vstes627:30244334
pythonvspyt875:30259475
pythonvsnew554cf:30281909
pythontb:30283811
wslfolderdoccf:30282075
vspre833:30267464
pythonptprofiler:30281270
vshan820cf:30276953
pythondataviewer:30285071
vscus158cf:30286554

```

</details>

**Comment by gjsjohnmurray** [comment]
Are you syncing settings between the two devices? If not, are your `scm.diffDecorations`* settings different between the two?

![image](https://user-images.githubusercontent.com/6726799/115133163-03407e80-9ffe-11eb-84d7-b84259e06968.png)

**Comment by nathanielfink** [comment]
Yes, they're synced. I've confirmed the settings in your image below:

[cid:ee4a9af3-62c9-4b79-9e1c-14ba7ef846a8]
________________________________
From: John Murray ***@***.***>
Sent: Sunday, 18 April 2021 13:25
To: microsoft/vscode ***@***.***>
Cc: Nathaniel Fink ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [microsoft/vscode] Changes not showing in gutter (#121564)


Are you syncing settings between the two devices? If not, are your scm.diffDecorations* settings different between the two?

[image]<https://user-images.githubusercontent.com/6726799/115133163-03407e80-9ffe-11eb-84d7-b84259e06968.png>

**Comment by ubaidjs** [comment]
facing same issue
